src/gui/autoexpandabledialog.cpp

```diff
@@ -65,6 +65,34 @@ QString AutoExpandableDialog::getText(QWidget *parent, const QString &title, con
     return d.m_ui->textEdit->text();
 }
 
+QString AutoExpandableDialog::getFileName(QWidget *parent, const QString &title, const QString &label,
```

Comment:
Instead of adding a new method, why not add an optional parameter to `getText`?
The parameter could be a pair of `int` which will feed to `textEdit->setSelection()`.

Reply:
Wouldn't a `bool excludeExtension` be enough? 
I think qbt should have this feature. Although i don't rename files in qbt, i _really_ appreciated this in other programs. Also should i push it here or make a new PR?